### DISCUSSION TITLE
New layout options in Preview 1.11.1_04

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#868] [Announcements]
In the latest preview, I have fixed some layout-related bugs and added new ways of creating layouts and changing node locations.

1. In addition to the distance between child nodes also the base distance from the parent node to the child node has been made configurable for parent nodes and for styles. Both distances can be changed using the formatting panel and using the moving handle with the pressed control/command button.
1. Node movements using the moving handle are restricted to a single direction. The vertical or horizontal direction is automatically chosen by the direction of the greater change. If the shift key is hold, moving the node in both directions simultaneously is possible as before.
1. The same logic is applied to changing the vertical and horizontal distances between the nodes using mouse with the pressed control/command button.
1. All selected nodes are synchronously moved in the direction from/to their parent node if multiple nodes are selected.

Please don't hesitate to share your positive or negative feedback.

**Comment by Illegit** [comment]
Hi Dimitry, I tried, but I couldn't drag the nodes with the mouse.

**Reply by dpolivaev** [reply]
What happens if you drag the handle shown to the side of the node?

<img width="40" alt="movingHandle" src="https://user-images.githubusercontent.com/1558052/206416674-cb187151-9181-43b8-aadb-349d4b811cc7.png">

**Reply by cesar404** [reply]
Dimitry, the handles are not shown anymore. Also the folding symbols don't appear when I hover over a parent node; the symbol only appears when the branch is already folded. Other than that, the new layouts look great! Thank you very much!

**Reply by abc16361** [reply]
> What happens if you drag the handle shown to the side of the node?
> 
> <img alt="movingHandle" width="40" src="https://user-images.githubusercontent.com/1558052/206416674-cb187151-9181-43b8-aadb-349d4b811cc7.png">

Dragging inside a node to move does not work. Instead dragging the handle horizontally and vertically works fine. I have made a bug report.

**Reply by dpolivaev** [reply]
Now I got it. I created a new preview version 1.11.1_05 with the fix

**Reply by cesar404** [reply]
> Now I got it. I created a new preview version 1.11.1_05 with the fix

It's working smoothly so far. Awesome features!

**Comment by Illegit** [comment]
And even it changed some of the nodes shapes

**Reply by dpolivaev** [reply]
I do not understand what your sentence means. If there are any unexpected changes don't hesitate to open an issue and don't forget to upload screenshots showing the changed shapes in the old and in the new version and the zipped mind map file for analysis.

**Reply by Illegit** [reply]
Sorry cannot answer effectively till this sunday.
